src/UIView/UIView+Constraints.m

```diff
@@ -0,0 +1,231 @@
+    NSLayoutConstraint *widthConstraint = [NSLayoutConstraint constraintWithItem:view
+                                                                       attribute:NSLayoutAttributeWidth
+                                                                       relatedBy:NSLayoutRelationEqual
+                                                                          toItem:nil
+                                                                       attribute:NSLayoutAttributeNotAnAttribute
+                                                                      multiplier:1.0
+                                                                        constant:width];
+    view.translatesAutoresizingMaskIntoConstraints = NO;
+    [self addConstraint:widthConstraint];
+}
+
+- (void)constrainView:(UIView *)view toHeight:(CGFloat)height {
+    NSLayoutConstraint *heightConstraint = [NSLayoutConstraint constraintWithItem:view
+                                                                        attribute:NSLayoutAttributeHeight
+                                                                        relatedBy:NSLayoutRelationEqual
+                                                                           toItem:nil
+                                                                        attribute:NSLayoutAttributeNotAnAttribute
+                                                                       multiplier:1.0
+                                                                         constant:height];
+    view.translatesAutoresizingMaskIntoConstraints = NO;
+    [self addConstraint:heightConstraint];
+}
+
+- (void)constrainView:(UIView *)view aboveView:(UIView *)positioningView {
+    [self constrainView:view attribute:NSLayoutAttributeBottom toView:positioningView attribute:NSLayoutAttributeTop];
+}
+
+- (void)constrainView:(UIView *)view belowView:(UIView *)positioningView {
+    [self constrainView:view attribute:NSLayoutAttributeTop toView:positioningView attribute:NSLayoutAttributeBottom];
+}
+
+- (void)constrainView:(UIView *)view leftOfView:(UIView *)positioningView {
+    [self constrainView:view attribute:NSLayoutAttributeRight toView:positioningView attribute:NSLayoutAttributeLeft];
+}
+
+- (void)constrainView:(UIView *)view rightOfView:(UIView *)positioningView {
+    [self constrainView:view attribute:NSLayoutAttributeLeft toView:positioningView attribute:NSLayoutAttributeRight];
+}
+
+- (void)constrainView:(UIView *)viewA attribute:(NSLayoutAttribute)attributeA toView:(UIView *)viewB attribute:(NSLayoutAttribute)attributeB {
+    [self constrainView:viewA
+              attribute:attributeA
+                 toView:viewB
+              attribute:attributeB
+               constant:0.0
+             multiplier:1.0];
+}
+
+- (void)constrainView:(UIView *)viewA attribute:(NSLayoutAttribute)attributeA toView:(UIView *)viewB attribute:(NSLayoutAttribute)attributeB constant:(CGFloat)constant multiplier:(CGFloat)multiplier {
```

Comment:
I don't think one has been specified for this project yet, is there a character limit in our general style guide?